maintenance/migratestatus/migrator/migrator.go

```diff
@@ -0,0 +1,281 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migrator
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/google/go-github/github"
+	"k8s.io/test-infra/maintenance/githubutil"
+)
+
+var (
+	stateAny     = "ANY_STATE"
+	stateSuccess = "success"
```

Comment:
This is variable is only used once. Please delete this variable and move the literal to where it is used.